benchttp/report.go

```diff
@@ -17,14 +16,14 @@ type Report struct {
 
 // Metadata contains contextual information about a run.
 type Metadata struct {
-	Config        config.Global
+	Config        Runner
```

Comment:
```suggestion
	Runner        Runner
```

Suggestion: let's go all the way in removing the notion of `Config` and call it `Runner`, aka "produced by this runner".

Reply:
Yup good catch!